UnitTests/expected/Strings-File-Dot-Syntax.swift.out

```diff
@@ -0,0 +1,70 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+import Foundation
+
+// swiftlint:disable file_length
+// swiftlint:disable type_body_length
+// swiftlint:disable nesting
+
+struct L10n {
+  /// Title of the alert
+  static let AlertTitle = L10n.tr("alert_title")
+  /// Some alert body there
+  static let AlertMessage = L10n.tr("alert_message")
+  /// Hello, my name is %@ and I'm %d
+  static func Greetings(p0: String, p1: Int) -> String {
+    return L10n.tr("greetings", p0, p1)
+  }
+
+  struct Apples {
+    /// You have %d apples
+    static func Count(p0: Int) -> String {
+      return L10n.tr("apples.count", p0)
+    }
+  }
+
+  struct Bananas {
+    /// Those %d bananas belong to %@.
+    static func Owner(p0: Int, p1: String) -> String {
+      return L10n.tr("bananas.owner", p0, p1)
+    }
+  }
+
+  struct Object {
+    /// These are %3$@'s %1$d %2$@.
+    static func Ownership(p0: Int, p1: String, p2: String) -> String {
+      return L10n.tr("object.ownership", p0, p1, p2)
+    }
+  }
+
+  struct Settings {
+
+    struct NavigationBar {
+
+      struct Title {
+
+        struct Even {
+          /// Settings
+          static let Deeper = L10n.tr("settings.navigation-bar.title.even.deeper")
```

Comment:
We should probably add another string which has more than 6 levels, so that we can write a Unit Test for such cases, and ensure it doesn't generate code that fails to compile if people happen to use deeper levels (but rather generate code that just stops splitting/nesting at level 6 and use `_` for higher level… but would at least compile)